### PR TITLE
[KO] Translation of Erlang Interoperability Lesson

### PR DESCRIPTION
**Pull request by Dalgona**
@devleoper, @malkoG, @riseshia
I am back! The final exam for this semester has just finished! :tada:
It's been a quite long time since I made this draft, so please read carefully while reviewing this.
Any corrections, improvements, suggestions, or any other comments are welcome. :rocket:

### CODE REVIEW

File: ko/lessons/advanced/erlang.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+```elixir
+png = :png.create(#{:size => {30, 30},
+                    :mode => {:indexed, 8},
+                    :file => file,
+                    :palette => palette}),
+```
+
+## 눈에 띄는 차이점
+
+Erlang을 어떻게 사용하는지 배웠기 때문에, 이제 Erlang 상호 운용을 하면서 딸려오는 실수들에 대해 알아야 합니다.
```
Comment (riseshia) [inline ko/lessons/advanced/erlang.md]: (maybe) `딸려오는 실수들에  대해 알아야 합니다.` -> `따라오는 실수들을 알아야 합니다`

**Comment by riseshia** [comment]
LGTM 👍

**Comment by Dalgona** [comment]
Thank you @riseshia! :rocket:

**Comment by malkoG** [comment]
Pretty nice! :100:

**Comment by devleoper** [comment]
💯

**Comment by devleoper** [comment]
Thank you, @dalgona!

**Pull request merged**